index.js

```diff
@@ -30,7 +30,9 @@ const pascalCase = (string) => {
  * @param {boolean} isApiMonolith - flag indicating whether to treat this api as monolithic.
  * @param {string} userProvidedServiceName - Optional service name for api file.
  */
-exports.generate = async (inputFile, outputDirectory, isApiMonolith, userProvidedServiceName) => {
+exports.generate = async (inputFile, outputDirectory, isApiMonolith, userProvidedServiceName, axiosVersion = 0) => {
+	// Axios header typings change at version 1.0. See https://github.com/axios/axios/blob/v1.x/CHANGELOG.md#100---2022-10-04
+	const useNewAxiosHeaderTypes = axiosVersion >= 1;
```

Comment:
this const name would be more self explanatory, thus not needing the comment, if it were changed to `isAxiosVersionZero: boolean` or `axiosVersion: number`

("is" being the preferred prefix in boolean variables in this repo, and type being number can be a more clear way to distinguish why a change to the template could occur)

Would then look something like:
IS_AXIOS_VERSION_ZERO: isAxiosVersionZero
...
{{#IS_AXIOS_VERSION_ZERO}}Record<string, unknown>{{/IS_AXIOS_VERSION_ZERO}}{{^IS_AXIOS_VERSION_ZERO}}RawAxiosRequestHeaders{{/IS_AXIOS_VERSION_ZERO}}```

This will inform why the new type will be infjected if looking only at the .mustache